README.md

```diff
@@ -126,14 +127,9 @@ Navigate to your `op-geth` directory and start service by running the command:
 
 ```sh
 ./build/bin/geth \
+    --op-network=$OP_NODE_NETWORK \
```

Comment:
With this change i think the section above [`Initialize op-geth`](https://github.com/LiskHQ/lisk-node/pull/48/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R110-R122) becomes redundant and could be removed.